### PR TITLE
Add Fedora installation instructions

### PR DESCRIPTION
**Pull request by hroncok**
cc @decathorpe

This is still in the process of being available, so don't merge yet, please. I'll mark this as ready for review once the command actually works on Fedora 34.

### CODE REVIEW

**Comment by hroncok** [comment]
The command now actually works on Fedora 34.

**Comment by brettcannon** [comment]
@hroncok I just got back from vacation, but this is in my review queue so I will get to it.

But I did want to ask you and @decathorpe if https://github.com/brettcannon/python-launcher/discussions/146 would make your lives easier by having a `.deb` config for something like https://github.com/mmstick/cargo-deb live in the repo itself? Or is it not really helpful to you in the end? The key thing for me is having someone help verify the file doesn't accidentally break since a `.deb` file isn't how I'm installing the Launcher for myself.

**Comment by decathorpe** [comment]
Our Rust packaging tools work almost exactly how cargo works without "special sauce" or configuration files (basically only "cargo install", with very basic configuration to tell it where to look for crate dependencies, and where to copy installed binaries), so adding configuration files to crate sources directly is not necessary for us.

Also we're dealing with RPM packages here, so I can't really help with deb stuff at all :)

**Comment by brettcannon** [comment]
Sorry about the distro confusion! Long day of catch-up after vacation 😅

File: README.md [hunk 1]
```diff
@@ -55,6 +55,16 @@ yay -S python-launcher
 
 https://aur.archlinux.org/packages/python-launcher
 
+#### [Fedora Linux](https://getfedora.org/)
```
Comment (brettcannon) [inline README.md:58]: This is already in the Linux section, so no need to specify that Fedora is a Linux distro.

```suggestion
#### [Fedora](https://getfedora.org/)
```

**Review by brettcannon — APPROVED** [review]

**Comment by brettcannon** [comment]
Thanks for getting this into Fedora!

File: README.md (hunk 1 above)
Comment (hroncok) [reply README.md]: Fedora Linux is the official name of the distro that Fedora produces. There has been some discussion around this in the Fedora community and the consensus is to call the distro Fedora Linux where possible. Relevant change proposal: https://fedoraproject.org/wiki/Changes/Fedora_Linux_in_os-release

**Pull request merged**